src/main/webapp/css/timeline.css

```diff
@@ -45,6 +47,66 @@
   text-align: left;
 }
 
+.timeline > li .timeline-panel{
+  background-color: rgba(255, 255, 255, 0.6);
+  border-radius: 0.5em;
+}
+
+.timeline .delete {
+  float: right;
+  margin-right: -19px;
+  font-size: 1.5em;
+  color: rgba(255, 146, 52, .8);
+  padding: 0;
+  background-color: rgba(255,255,255,0);
+  border-style: hidden;
+}
+
+.timeline .delete:hover, .timeline .delete:active{
+  color: rgba(229, 127, 39, 0.87);
+}
+
+.deleteConfirmation {
+  position: absolute;
+  width: 100%;
+  left: 0;
+  text-align: center;
+  transition: opacity 0.1s ease;
+  -webkit-transition: opacity 0.1s ease;
+  -moz-transition: opacity 0.1s ease;
+  font-family:sans-serif;
+  background-color:#29130dcf;
+  color:white;
+}
+
+.deleteConfirmation a {
+  color: white;
+  text-decoration: underline;
+  font-weight: bold;
+}
+
+.deleteConfirmation a:hover {
+  color: #c3bdbd;
+}
+
+.deleteConfirmation button {
+  color: white;
+}
+
+.deleteConfirmation button:hover {
+  color: white;
+}
+
+.deleteConfirmation.show {
+  z-index:101;
```

Comment:
Any reason for 101? Would just 1 work as well?

Reply:
The z-index of image is set to be 100. When the screen width gets small, it can overlaps the image.